cas-server-support-wsfederation-sample-site/src/main/webapp/WEB-INF/spring-configuration/wsfederation.xml

```diff
@@ -42,18 +42,18 @@
          attributeMutator: (optional) a class (defined by you) that can modify the attributes/assertions returned by the ADFS server
          signingCertificateFiles: ADFS's signing certificate used to validate the token/assertions issued by ADFS.
          -->
-        <property name="identityProviderIdentifier" value="http://adfs.example.org/adfs/services/trust" />
-        <property name="identityProviderUrl" value="https://adfs.example.org/adfs/ls/" />
-        <property name="identityAttribute" value="upn" />
-        <property name="relyingPartyIdentifier" value="urn:federation:cas" />
-        <property name="tolerance" value="60000" />
+        <property name="identityProviderIdentifier" value="${adfs.idp.id:http://iam-dev-windows.something.net/adfs/services/trust}" />

```

Comment:
I don't think almost any of these defaults make sense... They should be just the cas.properties properties... upn can probably stay, tolerance of 60000 can probably stay. Not sure about the signing cert.